keyboards/handwired/nortontechpad/info.json

```diff
@@ -0,0 +1,52 @@
+{
+    "keyboard_name": "NortonTechPad",
+    "manufacturer": "NortonTech",
+    "url": "",
+    "maintainer": "NortonTech",
+    "usb": {
+        "vid": "0xFEED",
+        "pid": "0x9801",
+        "device_version": "0.0.1"
+    },
+    "features": {
+        "bootmagic": true,
+        "extrakey": false,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["D7", "E6", "B4", "B5"],
+        "rows": ["F7", "B1", "B3", "B2", "B6"]
+    },
+    "diode_direction": "COL2ROW",
+    "development_board": "promicro",
+    "layouts": {
```

Comment:
```suggestion
    "community_layouts": ["numpad_5x4"],
    "layouts": {
```

Reply:
This keyboard does not use the community layout, the matrix/keymap will not work with this change

Reply:
If I change the to the community layout it does not type correctly